draft-ietf-webpush-protocol.xml

```diff
@@ -501,14 +499,20 @@ TTL = 1*DIGIT
         </t>
         <t>
           Absence of the TTL header field is interpreted as equivalent to a zero
-          value.  Push messages with a zero TTL indicate that storage is not
-          needed and that the message can be dropped if the user agent isn't
-          immediately available to receive the message.  Push messages with a
-          zero TTL can be delivered very efficiently.
-          <!-- But we need to determine if they can be acknowledged.  Decoupling
-               of delivery and acknowledgment makes this tricky. -->
+          value.  A Push message with a zero TTL is immediately delivered if the
+          user agent is available to receive the message.
         </t>
         <t>
+          The push service is then permitted to immediately remove the push message
+          resource which might occur before the user agent acknowledges receipt of the
+          message by performing a HTTP DELETE on the resource. Consequently, an
+          application server cannot rely on receiving acknowledgement receipts for zero
+          TTL push messages.
+        </t>
+        <t>
+          If the user agent is unavailable, the message expires and is never delivered.

```

Comment:
Again, this needs to identify more clearly that it applies to zero RTT messages.